.circleci/config.yml

```diff
@@ -1043,12 +1048,14 @@ workflows:
               only:
                 - release
                 - master
+                - headless-staging
                 - beta
-      - api_2-amplify_e2e_tests:
+      - api_1-amplify_e2e_tests:
```

Comment:
Are these auto-changes?

Reply:
Yeah this was changed by the split test script.